tests/integration/records/serializers/json/test_jobs.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import json
+from copy import deepcopy
+
+from invenio_accounts.testutils import login_user_via_session
+from marshmallow import utils
+
+from inspirehep.accounts.roles import Roles
+
+
+def test_jobs_json(api_client, db, create_record_factory, datadir):
+    headers = {"Accept": "application/json"}
+
+    data = json.loads((datadir / "955427.json").read_text())
+
+    record = create_record_factory("job", data=data)
```

Comment:
```suggestion
    record = create_record("job", data=data)
```

Reply:
I don't wanna change everything to `created_record` because stuff starts failing and takes time to fix, let's do it after this one.